src/client_impl_test.cc

```diff
@@ -208,14 +248,24 @@ TEST_F(MixerClientImplTest, TestSuccessCheckAndQuota) {
   EXPECT_TRUE(done_status.ok());
 
   for (int i = 0; i < 10; i++) {
-    // Other calls should ba cached.
+    // Other calls should be cached.
     Status done_status1 = Status::UNKNOWN;
     client_->Check(request_, quotas_, empty_transport_,
                    [&done_status1](Status status) { done_status1 = status; });
     EXPECT_TRUE(done_status1.ok());
   }
   // Call count should be less than 4
   EXPECT_LE(call_counts, 3);
+  Statistics stat;
+  client_->GetStatistics(&stat);
+  // Less than 4 remote calls are made for prefetching, and they are
+  // non-blocking remote calls.
+  EXPECT_EQ(11, stat.total_check_calls);
+  EXPECT_GE(3, stat.total_remote_check_calls);
```

Comment:
remote_check_calls should be <=3, same as line 258

Reply:
Done.